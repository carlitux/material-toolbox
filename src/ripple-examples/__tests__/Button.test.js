// @flow
import React from 'react';
import { shallow } from 'enzyme';

import ButtonRipple from '../Button';

describe('example::ButtonRipple', () => {
  it('Should match snapshoot', () => {
    const component = shallow(<ButtonRipple />);
    expect(component).toMatchSnapshot();
  });
});
