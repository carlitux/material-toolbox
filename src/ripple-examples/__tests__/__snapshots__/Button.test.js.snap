// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`example::ButtonRipple Should match snapshoot 1`] = `
<Inner>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';
import Elevation from 'material-toolbox/elevation';

const component = (
<Elevation depth={2}>
  <Ripple>
    <button type=\\"button\\" className=\\"ripple-demo-surface\\">
      button
    </button>
  </Ripple>
</Elevation>
)"
      title="Applied to <button> element"
    >
      <Elevation
        depth={2}
        transition={false}
      >
        <Ripple
          accent={false}
          cssOnly={false}
          disabled={false}
          primary={false}
          unbounded={false}
        >
          <button
            className="ripple-demo-surface"
            type="button"
          >
            button
          </button>
        </Ripple>
      </Elevation>
    </Example>
  </Cell>
  <Cell
    desktop={6}
  >
    <Example
      source="import Ripple from 'material-toolbox/ripple';
import Elevation from 'material-toolbox/elevation';

const component = (
<Elevation depth={2}>
  <Ripple cssOnly>
    <button type=\\"button\\" className=\\"ripple-demo-surface\\">
      button
    </button>
  </Ripple>
</Elevation>
)"
      title="Applied to <button> element - CSS Only"
    >
      <Elevation
        depth={2}
        transition={false}
      >
        <Ripple
          accent={false}
          cssOnly={true}
          disabled={false}
          primary={false}
          unbounded={false}
        >
          <button
            className="ripple-demo-surface"
            type="button"
          >
            button
          </button>
        </Ripple>
      </Elevation>
    </Example>
  </Cell>
</Inner>
`;
